apps/zotonic_launcher/src/command/zotonic_cmd_addsite.erl

```diff
@@ -192,11 +192,20 @@ parse_args([ "-n", Schema | Args ], Acc) ->
     parse_args(Args, Acc#{ dbschema => Schema });
 parse_args([ "-a", Pw | Args ], Acc) ->
     parse_args(Args, Acc#{ admin_password => Pw });
-parse_args([ "-A", "true" | Args ], Acc) ->
-    parse_args(Args, Acc#{ app => true });
-parse_args([ "-A", "false" | Args ], Acc) ->
-    parse_args(Args, Acc#{ app => false });
+parse_args([ "-A", App | Args ], Acc) ->
```

Comment:
Interesting approach. You can try simplify it. I suppose expected default value `false` for `app`. If yes, then it can be done like:

```erlang
parse_args([ "-A", App | Args ], Acc) ->
   parse_args(Args, Acc#{ app => app_to_boolean(string:to_lower(App)) });
%....

app_to_boolean(App) when App =:= "true"; App =:= "1" -> true;
app_to_boolean(_) -> false.
```

Reply:
Hm.. I see function `z_convert:to_bool/1`, maybe you can try reuse it :thinking:?
```erlang
parse_args([ "-A", App | Args ], Acc) ->
    parse_args(Args, Acc#{ app => `z_convert:to_bool(string:to_lower(App)) });
```

Reply:
Good catch! `z_convert:to_bool/1` works.